folium/plugins/beautify_icon.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         inner_icon_style="",
         spin=False,
         number=None,
-        **kwargs
+        **kwargs,
```

Comment:
They were added by `ruff format`. I agree they do not make sense in this case. As far as I know there is no option to remove these trailing commas. I can manually undo these kind of changes, but its a hassle. I'd prefer to just blindly follow `ruff`.

Reply:
We don't run `ruff format` on this repo yet though. We use Black to autoformat and have Ruff as linter. So it's better not to run `ruff format`, since it will create a lot of changes.

Reply:
We could switch over to `ruff format` though and ditch Black. Maybe that's good to do anyway, and if we merge that before this PR that also solves the issue.

Reply:
My bad, I thought we used ruff as a formatter as well. I can create an extra PR to ditch black if you want, but I have no strong opinion either way. If not I will undo the ruff formatting changes.